Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml

```diff
@@ -0,0 +1,27 @@
+- type: entity
+  name: Brass knuckles
+  parent: ClothingHandsGlovesBoxingRed
+  id: WeaponBrassKnuckles
+  description: Ever wanted to punch someone in the mouth harder then regular? Well S.E.S.W.C has what you need, take your anger out with fashion on your enemies
+  components:
+  - type: Sprite
+    sprite: _NF/Objects/Weapons/Melee/brassknuckles.rsi
+    state: icon
+  - type: Clothing
+    sprite: _NF/Objects/Weapons/Melee/brassknuckles.rsi
+  - type: MeleeWeapon
+    autoAttack: true
+    attackRate: 2
+    damage:
+      types:
+       Blunt: 10
```

Comment:
it doesnt do 20 damage for second because everything has armor most common have  30 - 60 percent.  not to mention try fighting any of the PVE creatures  like  syndis/mercs/robots.  with brass knuckles most likely you would be near dead or dead

Reply:
As an apples-to-apples comparison, considering something's base values is fine, and relatively speaking, this is still powerful.

Reply:
it isnt an apples to apples comparison. in order to get close to the actual damage value I am trying to produce you have to increase the damage

Reply:
Perhaps melee weapons should be buffed, but standard fare brass knuckles should not be an _unusually strong_ melee weapon.